### PR TITLE
creating sandbox launch blog post

### PR DESCRIPTION
**Pull request by stefi-says**

### CODE REVIEW

**Review by sarob — APPROVED** [review]
I think this works for me. I updated the sandbox dochttps://github.com/OpenDataforWeb3/Resources/pull/130 and the miro board https://miro.com/app/board/uXjVPlv-XQA=/ to match you post

**Comment by G-r-ay** [comment]
I'm completely on board with it.✅
But I'll still the PR open so other set of eyes could have a look👀

File: content/en/blog/the_sandbox_launch.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- Featured inclusion in the [ODC Landscape](https://opendatacommunity.org/docs/landscape/)
+- Featured inclusion in our data stack   
+
+**Technical help**
+- Mentorship   
+- Feature in Hackathons   
+- Ongoing hackathon - aka bounties to accelerate and attract contributions  
+
+**Network**  
+- Community builders engagement and contributions (including some FDD/Gitcoin members)
```
Comment (Razga) [inline content/en/blog/the_sandbox_launch.md:44]: -people that have a passion for an open data and open internet future

Comment (Razga) [inline content/en/blog/the_sandbox_launch.md:44]: replace with -community builders and contributors passionate about an open data future

File: content/en/blog/the_sandbox_launch.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- Featured inclusion in our data stack   
+
+**Technical help**
+- Mentorship   
+- Feature in Hackathons   
+- Ongoing hackathon - aka bounties to accelerate and attract contributions  
+
+**Network**  
+- Community builders engagement and contributions (including some FDD/Gitcoin members) 
+- Other DAOs core members
```
Comment (Razga) [inline content/en/blog/the_sandbox_launch.md:45]: possibly replace with -other prolific figures from web3 organizations, DAOs and ofc independent thinkers

File: content/en/blog/the_sandbox_launch.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+In the coming weeks, the community will discuss which will be the initial projects invited to join the Sandbox, 
+these projects will be able to rely on numerous benefits such as:
+
+**Tooling**
+- Repositories inside ODC github   
+- Dedicated Discord channel  
+- Educational help on setting tools up   
+  
+**Promotion**   
+- web site listings
```
Comment (G-r-ay) [inline content/en/blog/the_sandbox_launch.md]: *Website

File: content/en/blog/the_sandbox_launch.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+description: "OpenData Community Hackathon: Partners host educational Twitter Spaces"
+lead: "Fight Sybils, Decentralize Data"
+contributors: ["Stefi"]
+date: 2023-02-19T22:26:54+01:00
+lastmod: 2023-02-19T22:26:54+01:00
+draft: false
+images: []
+---
+
+Our last hackathon occurred from Jan 5th till Jan 31st , competitors collaborated to 4 different bounties, each with a different focus like helping the community growth, developing dashboards,
```
Comment (Razga) [inline content/en/blog/the_sandbox_launch.md:12]: As you may know our last hackathon took place from Jan 5th till Jan 31st and competitors could start projects in 4 different areas(bounties), each with a different focus:
- developing anti sybil- dashboards 
- hacking the hackaton by helping the community grow/ creating educational materials(such as FAQs)
- identifying sybil behaviors through exploratory data analysis 
- packaging algorithms in “Legos” that can be chained together with the intention to collaborate for an accurate classification of sybil wallets 
More details about the competition can be found in this [post](https://opendatacommunity.org/blog/data-builder-hackathon/) about the hackathon opening.

File: content/en/blog/the_sandbox_launch.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+identifying sybil behaviors through exploratory analysis and packaging algorithms in “Legos” that can be chained together with the intention to collaborate 
+for an accurate classification of sybil wallets.  Details about the competition can be found in this [post](https://opendatacommunity.org/blog/data-builder-hackathon/) about the hackathon opening.
```
Comment (Razga) [inline content/en/blog/the_sandbox_launch.md]: Edit to 12-14:

As you may know our last hackathon took place from Jan 5th till Jan 31st and competitors could start projects in 4 different areas(bounties), each with a different focus:
- developing anti sybil- dashboards 
- hacking the hackaton by helping the community grow/ creating educational materials(such as FAQs)
- identifying sybil behaviors through exploratory data analysis 
- packaging algorithms in “Legos” that can be chained together with the intention to collaborate for an accurate classification of sybil wallets 
More details about the competition can be found in this [post](https://opendatacommunity.org/blog/data-builder-hackathon/) about the hackathon opening.

File: content/en/blog/the_sandbox_launch.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+identifying sybil behaviors through exploratory analysis and packaging algorithms in “Legos” that can be chained together with the intention to collaborate 
+for an accurate classification of sybil wallets.  Details about the competition can be found in this [post](https://opendatacommunity.org/blog/data-builder-hackathon/) about the hackathon opening.  
+
+We had an intense month with projects feedback sections together with FDD analysts, twitter spaces with sponsors and a lot more
+. What about the results? Incredible! We had 53 submissions and 29 winning projects that shared 40K in prizes, the projects range from python packages to 
+whole decentralized data frameworks that can integrate live dashboards, details and other results can be here on this [hackathon wrap up presentation](https://docs.google.com/presentation/d/1Ic4p9o2pLHETjBv1N9v4wkLTYToZxh-GP6Bo8ICtkko/edit#slide=id.g20c507cff6c_0_1)
+Most important of all, some projects developed in the hackathon demonstrate great value, scalability and potential to be strong weapons in the fight against sybil 
+attacks, mainly in the distribution of public funds.  
+
```
Comment (Razga) [inline content/en/blog/the_sandbox_launch.md:27]: edit to 16-20:

We are proud to say that we've had an intense month filled with project feedback sessions(together with FDD analysts and other experts), twitter spaces with our sponsors and guests all while running the actual hackaton itself.

So...What about the results? They were incredible! We had 53 submissions in total with 29 projects that won and shared 40K in prizes! The projects range from analysis tools to python packages to 
whole decentralized data frameworks that can integrate live dashboards. More details and the hackaton results can be here on this [hackathon wrap up presentation](https://docs.google.com/presentation/d/1Ic4p9o2pLHETjBv1N9v4wkLTYToZxh-GP6Bo8ICtkko/edit#slide=id.g20c507cff6c_0_1)
The best part is that besides the fact that a lot of people participated in the hackaton and submitted great projects, a lot of those projects demonstrated great value, scalability and potential to be strong weapons in the fight against sybil 
attacks, mainly in the distribution of public funds.

File: content/en/blog/the_sandbox_launch.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+- Ongoing hackathon - aka bounties to accelerate and attract contributions  
+
+**Network**  
+- Community builders engagement and contributions (including some FDD/Gitcoin members) 
+- Other DAOs core members   
+- Possible sponsors  
+
+To follow the development of the Sandbox keep an eye on our [ODC Sandbox github repo](https://github.com/OpenDataforWeb3/sandbox) and our channels, 
+till 1st of April we will already have projects set to start development and accept contributions. Join us!   
+
```
Comment (Razga) [inline content/en/blog/the_sandbox_launch.md]: edit to 22-49:

Therefore, we are very pleased to announce the birth of the OpenData Community Sandbox, a space to foster the maturation of these projects.
In the coming weeks, the community will discuss which projects will be the initial ones invited to join the Sandbox, these projects will be able to rely on numerous benefits such as:

**Tooling**
- Repositories inside ODC github   
- Dedicated Discord channel  
- Educational help on setting tools up   
  
**Exposure**   
- Web site listings   
- Twitter spaces   
- Ongoing promotion through ODC channels   
- Featured inclusion in the [ODC Landscape](https://opendatacommunity.org/docs/landscape/)
- Featured inclusion in our data stack   

**Technical help**
- Mentorship   
- Feature in Hackathons   
- Bounties to accelerate and attract contributions  

**Network**  
- People that have a passion for open data and an open internet
- Community builders and contributtors (including some FDD/Gitcoin members) 
- Various DAO core members and some prolific figures from web3
- Possible sponsors

To follow the development of the Sandbox keep an eye on our [ODC Sandbox github repo](https://github.com/OpenDataforWeb3/sandbox) and our channels. 
Our goal is to have projects set to start development and accept contributions until the 1st of April. Join us in building our open data dreams!

File: content/en/blog/the_sandbox_launch.md (hunk 3 above)
Comment (stefi-says) [reply content/en/blog/the_sandbox_launch.md]: adjusted in the new commit, thanks

File: content/en/blog/the_sandbox_launch.md (hunk 5 above)
Comment (stefi-says) [reply content/en/blog/the_sandbox_launch.md]: adjusted in the new commit, thanks!

File: content/en/blog/the_sandbox_launch.md (hunk 7 above)
Comment (stefi-says) [reply content/en/blog/the_sandbox_launch.md]: adjusted in the new commit, thanks

**Comment by stefi-says** [comment]
It is still missing a cover image, would love suggestions

**Comment by epowell101** [comment]
Looking good.  In the ending we write “till April 1st”…. I think we mean that we will have an operational Sandbox program in place BY April 1st.

As a call to action at the end instead of it mostly being - watch this space - what if we were to ask potential sandbox projects to get in touch?

Something like: “Projects and teams considering launching projects that might be a good fit for the OpenData Community sandbox, please reach out.  We are looking to support projects that help achieve our dual mission of protecting web 3 via resisting centralization at the data layer and by identifying and countering Sybil and other attackers. “

**Comment by sarob** [comment]
> Looking good. In the ending we write “till April 1st”…. I think we mean that we will have an operational Sandbox program in place BY April 1st.
> 
> As a call to action at the end instead of it mostly being - watch this space - what if we were to ask potential sandbox projects to get in touch?
> 
> Something like: “Projects and teams considering launching projects that might be a good fit for the OpenData Community sandbox, please reach out. We are looking to support projects that help achieve our dual mission of protecting web 3 via resisting centralization at the data layer and by identifying and countering Sybil and other attackers. “

I’d incorporate @epowell101 recommendations and push to publish. 

For a CC licensed image I like https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.F_Z2MPxMIU8Iy03BOq0MyQHaE6%26pid%3DApi&f=1&ipt=ed26ccfd9897c2afe067972be9c3b7bff4d085872f477cbbf0b6bf7f6f27d83d&ipo=images , https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.S3nkBFYSc7b_yp6MNe-fcAHaE6%26pid%3DApi&f=1&ipt=2ee6a0117d9ca719d1aa91c849d6c5c91da89e34d7e24eeee8091b7dbe47bbd2&ipo=images , or https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.Ez3jB1zcc7rxU72iXtPdrgHaFK%26pid%3DApi&f=1&ipt=af73bb9d6bae1a0ace748a2dee07237064617fec6d1c7a1a1bc7838ccc89577a&ipo=images

**Pull request merged**